codecept.conf.js

```diff
@@ -61,6 +61,7 @@ exports.config = {
     enterRespondentsPage: './e2e/pages/enterRespondents/enterRespondents.js',
     enterOthersPage: './e2e/pages/enterOthers/enterOthers.js',
     ordersNeededPage: './e2e/pages/ordersNeeded/ordersNeeded.js',
+    enterFamilyManPage: './e2e/pages/enterFamilyMan/enterFamilyMan.js'
```

Comment:
Please add `,` at the end of the line.

```suggestion
    enterFamilyManPage: './e2e/pages/enterFamilyMan/enterFamilyMan.js',
```